documentation/API_documentation/QoD_User_Story.md

```diff
@@ -1,10 +1,10 @@
-# QoD Stable Bandwidth and Latency API User Story
+# Quality-On-Demand (QoD) API User Story
 
 | **Item** | **Details** |
 | ---- | ------- |
-| ***Summary*** | As an application developer belonging to an enterprise, I want to request (using my application server/backend service), stable bandwidth/latency for a specified App-Flow from a Communication Service Provider (CSP), so that I can ensure better quality of experience for our end users inlcuding under network congestion.  |
-| ***Roles, Actors and Scope*** | **Roles:** Customer:User<br> **Actors:** Application service providers, hyperscalers, application developers. Currently the API does not explicitly handle consent management (expects that the customer either owns the device subscriptions or has agreed to manage consent as a part of general Terms and Conditions) and hence end users are not included as Actors. As this changes, the actor list will be extended with end users. <br> **Scope:** *Order To Activate (OTA)* \- Create/Remove/Get service instance (QoD session)\, get notification about the instance (QoD session) |
+| ***Summary*** | As an application developer belonging to an enterprise, I want to request (using my application server/backend service), stable latency or throughput for a specified App-Flow from a Communication Service Provider (CSP), so that I can ensure better quality of experience for our end users inlcuding under network congestion.  |
```

Comment:
```suggestion
| ***Summary*** | As an application developer belonging to an enterprise, I want to request (using my application server/backend service), stable latency or prioritized throughput for a specified App-Flow from a Communication Service Provider (CSP), so that I can ensure better quality of experience for our end users including under network congestion.  |
```

Reply:
I would rather make it explicit into "stable latency or stable bandwidth". But prioritization is just one technical mean to achieve that (to the extent which is possible, yes, not guaranteed).

Reply:
Partly done in [7bd2faa](https://github.com/camaraproject/QualityOnDemand/pull/103/commits/7bd2faa26ded46e2edf008822349fd41a5707451)

Reply:
I don't think we should talk explicitly about stable throughput or bandwidth at this point. In the introduction in the [API doc](https://github.com/camaraproject/QualityOnDemand/blob/main/documentation/API_documentation/QoD_API.md), we already use the expression "prioritized throughput": "Being able to request a stable latency (reduced jitter) or **prioritized throughput** from the network can improve user experience substantially.". "prioritized traffic" would serve as well.

There's still a minor typo at "inlcluding".

Reply:
Sorry about the typo ... I will make a short commit.

But the documentation isn't consistent regarding "prioritized":
* "to request stable latency or throughput" (in first paragraph)
* "request a stable latency (reduced jitter) or prioritized throughput" (first paragraph of intro)
* "prioritized service with stable latency or throughput" (within detail)
... the rest is all "(stable) latency or throughput"

For me "prioritized service with stable latency or throughput" is the most correct.

Reply:
I agree it is not consistent. In the discussions for the documentation we made some effort with the wording in order to avoid expectancy for a guaranteed service in terms of latency or throughput. It is not easy. Probably "prioritized service with stable latency or throughput" is the most correct to me also.